aws_datadog_iam.tf

```diff
@@ -0,0 +1,108 @@
+data "aws_iam_policy_document" "datadog_aws_integration_assume_role" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type = "AWS"
+      identifiers = ["arn:aws:iam::464622532012:root"] # Datadog AWS Account ID
+    }
+    condition {
+      test = "StringEquals"
+      variable = "sts:ExternalId"
+
+      values = [
+        "${var.datadog_aws_integration_external_id}"
+      ]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "datadog_aws_integration" {
+  statement {
+    actions = [
+        "apigateway:GET",
```

Comment:
@quangnhut123 is it better to refactor this list as a variable to let ppl easy config which ones they would like to monitor ?